src/panels/config/zwave/zwave-network.ts

```diff
@@ -59,136 +56,136 @@ class ZwaveNetwork extends PolymerElement {
         <ha-card class="content">
           <div class="card-actions">
             <ha-call-service-button
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="add_node_secure"
             >
               Add Node Secure
             </ha-call-service-button>
             <ha-service-description
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="add_node_secure"
-              hidden$="[[!showDescription]]"
+              .hidden="${!this._showHelp}"
             >
             </ha-service-description>
 
             <ha-call-service-button
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="add_node"
             >
               Add Node
             </ha-call-service-button>
             <ha-service-description
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="add_node"
-              hidden$="[[!showDescription]]"
+              .hidden="${!this._showHelp}"
             >
             </ha-service-description>
 
             <ha-call-service-button
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="remove_node"
             >
               Remove Node
             </ha-call-service-button>
             <ha-service-description
-              hass="[[hass]]"
+              .hass="${this.hass}"
               domain="zwave"
               service="remove_node"
-              hidden$="[[!showDescription]]"
+              .hidden="${!this._showHelp}"
```

Comment:
This is not equivalent. If you notice in the Polymer code, it's `hidden$`. That meant that hidden was set as an attribute. If you want to mimic that in Lit, use `?` to toggle attribute if value is true or false:

```suggestion
              ?hidden=${!this._showHelp}
```

Reply:
Alternatively, you can also just completely remove the element from the DOM if it's hidden.

```ts
return html`
${this._showHelp ? 
  html`<ha-service-description></ha-service-description>` : ""
}
`
```